install/update_91_92.php

```diff
@@ -772,6 +772,66 @@ function update91to92() {
                                  true);
    }
 
+   // add projecttemplate
+   $migration->addField("glpi_projects", "projecttemplates_id", "integer");
+   $migration->addField("glpi_projects", "is_template", "bool");
+   $migration->addField("glpi_projects", "template_name", "string");
+   $migration->addKey("glpi_projects", "projecttemplates_id");
+
+   $migration->addField("glpi_projecttasks", "projecttemplates_id", "integer");
```

Comment:
you missed ``projecttasktemplates_id`` in update script, maybe a typo in projecttemplates_id